web/src/routes/(app)/account/jobs/+page.svelte

```diff
@@ -168,6 +230,23 @@
 									</div>
 								</div>
 							</td>
+							<td class="align-top px-3 py-4 flex justify-end">
+								<button 
+									class="hover:text-red-500"
+									title="Remove this job"
+									on:click={handleJobRemoval(job.job_id)}
```

Comment:
We only want to show this button for non-verified jobs. Keep the column though because we are going to add additional actions soon (Interested, Not Interested, Save for Later)